Xamarin.Forms.Core/ReflectionExtensions.cs

```diff
@@ -1,32 +1,38 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
 using System.Reflection;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
-	internal static class ReflectionExtensions
+	public static class ReflectionExtensions
```

Comment:
I think this is only used by the Binding system and the Xaml parser. no need to open it. everyone can rewrite the same